keps/0001a-meta-kep-implementation.md

```diff
@@ -0,0 +1,133 @@
+---
+kep-number: 1a
+title: Meta KEP Implementation
+authors:
+  - "@justaugustus"
+  - "@calebamiles"
+  - "@jdumars"
+owning-sig: sig-pm
+participating-sigs:
+  - sig-architecture
+reviewers:
+  - "@idvoretskyi"
+  - "@timothysc"
+approvers:
+  - "@bgrant0607"
+  - "@jbeda"
+editor:
+creation-date: 2018-09-08
+last-updated: 2018-09-16
+status: provisional
+see-also:
+  - KEP-1
+replaces:
+superseded-by:
+---
+
+# Meta KEP Implementation
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Why another KEP?](#why-another-kep)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Implementation Details / Notes / Constraints](#implementation-details--notes--constraints)
+    - [Define](#define)
+    - [Organize](#organize)
+    - [Visibility and Automation](#visibility-and-automation)
+    - [Constraints](#constraints)
+- [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+
+## Summary
+
+Drive KEP adoption through improved process, documentation, visibility, and automation.
+
+## Motivation
+
+The KEP process is the standardized structure for proposing changes to the Kubernetes project.
+
+In order to graduate KEPs to GA, we must iterate over the implementation details.
+
+This KEP seeks to define actionable / delegable items to move the process forward.
+
+Finally, by submitting a KEP, we gain an opportunity to dogfood the process and further identify areas for improvement.
+
+### Why another KEP?
+
+As [KEP-1] is currently de facto for the project, we must be careful to make changes to it in an iterative and atomic fashion.
+
+When proposing a KEP, we action over some unit of work, usually some area of code.
+
+In this instance, we treat [KEP-1] as the unit of work. That said, this would be considered a meta-KEP of the meta-KEP.
+
+### Non-Goals
+
+- API Review process
+- Feature request triage
+- Developer guide
+
+## Proposal
+
+### Implementation Details / Notes / Constraints
+
+#### Define
+
+- Refine existing KEP documentation
+- Define KEP [DACI]
+
+#### Organize
+
+- Move KEPs into [k/features]
+- Move design proposals into [k/features]
+- Move architectural documents into [k/features]
+- Deprecate design proposals
+- Rename [k/features] to [k/keps]
+- Enable redirects to [k/keps]
+- Prevent new KEPs and design proposals from landing in [k/community]
+- Correlate existing features with KEPs
+- Fix [KEP numbering races] by adopting one of two schemes:
+  - Issue number of the KEP tracking issue
+  - PR number for the PR that marks the KEP as `accepted`
+
+#### Visibility and Automation
+
+- Create tooling to present:
+  - KEPs, through some easy to use mechanism e.g., https://keps.k8s.io
+  - Project tracking across SIGs
+  - SIGs roadmaps
```

Comment:
This might be a bit orthogonal imo.  SIGs react on the fly.

Reply:
_Currently_, SIGs react on the fly. 
Ideally, moving forward, SIGs should be able to plan and provide some level of soft commit prior to the release cycle, and that information should be aggregatable through the metadata in each KEP. The shape of how exactly to aggregate the data hasn't been defined yet, though.

Reply:
Without specifics, I would remove.

Reply:
@timothysc -- removed